app/views/layouts/jisc_mail.html.erb

```diff
@@ -0,0 +1,5 @@
+<html>
+  <body>
+    <%= yield %>
```

Comment:
I don't think this file is necessary since JSIC mail command have to be in plaintext emails anyway.